README.md

```diff
@@ -0,0 +1,36 @@
+# Design first page  portfolio app based in a figma template 
```

Comment:
Please use a short name that defines your project e.g. `Portfolio` or `My Portfolio` as described in the Microverse [Readme Template](https://github.com/microverseinc/readme-template)😄

Reply:
- [ ] [OPTIONAL] Kindly implement this as required by the first reviewer but it should be optional cause is not part of the requirement 
> Please use a short name that defines your project e.g. Portfolio or My Portfolio as described in the Microverse [Readme Template](https://github.com/microverseinc/readme-template)smile